PluginCore/DockPanelSuite/Docking/DockPanel.FocusManager.cs

```diff
@@ -416,13 +416,13 @@ internal void SetActiveContent()
                     return;
 
                 if (m_activeContent != null)
-                    m_activeContent.DockHandler.IsActivated = false;
+                    m_activeContent.DockHandler.SetIsActivated(false);
```

Comment:
I haven't yet compared with the latest DPS.

Side note is that, the choice to introduce a method SetIsActivated() instead of a setter for IsActivated was to follow the other conventions in the package - like the dock panel had IsActivated getter and SetIsActivated().